modules/order/templates/commerce-order-total-summary.html.twig

```diff
@@ -0,0 +1,31 @@
+{#
+/**
+ * @file
+ * Default order total summary template.
+ *
+ * Available variables:
+ * - attributes: HTML attributes for the wrapper.
+ * - totals: An array of order totals values with the following keys:
+ *   - subtotal: \Drupal\commerce_price\Price|null The order subtotal amount.
+ *   - adjustments: array Adjustments groups with the following structure:
+ *     - <type>: An array keyed by adjustment type id.
+ *       - 'label': string The adjustment type label.
+ *       - 'items': An associative array of items, grouped by type and source id
+ *         if a source id exists, otherwise a numerically indexed item, each
+ *         with the following keys:
+ *         - 'amount': \Drupal\commerce_price\Price The total
+ *         - 'label': string The label of the adjustment or adjustment type.
+ *   - total: \Drupal\commerce_price\Price|null The order total amount.
+ *
+ * @ingroup themeable
+ */
+#}
+<div{{ attributes }}>
+  {{ 'Subtotal'|t }}: {{ totals.subtotal|commerce_price_format }}<br/>
+  {% for group in totals.adjustments %}
+    {% for item in group.items %}
+      {{ item.label }}: {{ item.amount|commerce_price_format }}<br/>
```

Comment:
We should do some basic `div` splitting it up, or something inline. Spans? so it can be easily styled w/o overriding